app/views/homepage/index.html.erb

```diff
@@ -29,7 +29,7 @@
   <hr>
 
   <section>
-    <h2>Prémiové dáta a API</h2>
+    <h2>Prémiové dáta & API</h2>
```

Comment:
nechal by som povodnu spojku... alebo co sa pouziva inde?

```suggestion
    <h2>Prémiové dáta a API</h2>
```

Reply:
kusok dole pod tym je pouzity "&", tak som chcel nech to je jednotne, pozri tu
 [Premiove data a API vs. Otvorene data & API ](https://ekosystem.slovensko.digital/)

Reply:
ok, beriem